src/zino/snmp.py

```diff
@@ -161,21 +192,17 @@ async def walk(self, *oid: str) -> list[MibObject]:
         """
         results = []
         current_object = self._oid_to_object_type(*oid)
-        try:
-            self._resolve_object(current_object)
-        except MibNotFoundError as error:
-            _log.error("%s: %s", self.device.name, error)
-            return results
+        self._resolve_object(current_object)
         original_oid = OID(str(current_object[0]))
         while True:
             current_object = await self._getnext(current_object)
-            if not current_object or not original_oid.is_a_prefix_of(str(current_object[0])):
+            if not original_oid.is_a_prefix_of(str(current_object[0])):
                 break
             mib_object = self._object_type_to_mib_object(current_object)
             results.append(mib_object)
         return results
 
-    async def getbulk(self, *oid: str, max_repetitions: int = 1) -> list[MibObject]:
+    async def getbulk(self, *oid: str, max_repetitions: int = 10) -> list[MibObject]:
```

Comment:
Why was this 1 to begin with, and why did it change to 10 in a PR focused on error handling?

Reply:
In order to avoid confusion: For bulk requests, I think generally a default value higher than 1 makes a lot of sense (the whole point of the bulk request is normally to fetch more than just 1 value!).

So I think we *should* increase the number.  I just don't think it's relevant for this specific PR.